aspnet-core/src/AbpCompanyName.AbpProjectName.Application/MultiTenancy/Dto/TenantDto.cs

```diff
@@ -5,18 +5,18 @@
 
 namespace AbpCompanyName.AbpProjectName.MultiTenancy.Dto
 {
-    [AutoMapTo(typeof(Tenant)), AutoMapFrom(typeof(Tenant))]
+    [AutoMapFrom(typeof(Tenant))]
```

Comment:
`[AutoMap(typeof(Tenant))]`

Reply:
Actually, the AutoMapTo is not used, as that mapping is done manually in TenantAppService (see https://github.com/aspnetboilerplate/module-zero-core-template/blob/master/aspnet-core/src/AbpCompanyName.AbpProjectName.Application/MultiTenancy/TenantAppService.cs#L98)